code/game/machinery/computer/computer.dm

```diff
@@ -83,24 +79,19 @@
 	update_icon()
 	return
 
-/obj/machinery/computer/proc/set_broken()
-	if(circuit) //no circuit, no breaking
-		stat |= BROKEN
-		update_icon()
-	return
-
 /obj/machinery/computer/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/weapon/screwdriver) && circuit && !(flags&NODECONSTRUCT))
 		playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
 		user << "<span class='notice'> You start to disconnect the monitor...</span>"
 		if(do_after(user, 20/I.toolspeed, target = src))
+			deconstruction()

```

Comment:
To drop things that needs to be removed when deconstructing the computer (see pandemic computer)